source/tutorials/longitudinal.rst

```diff
@@ -0,0 +1,96 @@
+Performing longitudinal and paired sample comparisons with q2-longitudinal
+==========================================================================
+
+.. note:: This guide assumes you have installed QIIME 2 using one of the procedures in the :doc:`install documents <../install/index>`.
+
+This tutorial will demonstrate the various features of `q2-longitudinal`, a plugin that supports statistical and visual comparisons of longitudinal study designs and paired samples, to determine if/how samples change between observational "states". "States" will most commonly be related to time, and the sample pairs should typically consist of the same individual subject  observed at two different time points. For example, patients in a clinical study whose stool samples are collected before and after receiving treatment.
+
+"States" can also commonly be methodological, in which case sample pairs will usually be the same individual at the same time with two different methods. For example, q2-feature-classifier could compare the effects of different collection methods, storage methods, DNA extraction methods, or any bioinformatic processing steps on the feature composition of individual samples.
+
+In the examples below, we will use the `moving pictures tutorial data`_. First let's create a new directory and download the relevant tutorial data.
+
+.. command-block::
+   :no-exec:
+
+   mkdir longitudinal-tutorial
+   cd longitudinal-tutorial
+
+.. download::
+   :url: https://data.qiime2.org/2017.7/tutorials/moving-pictures/sample_metadata.tsv
+   :saveas: moving-pictures-sample-metadata.tsv
+
+.. download::
+   :url: https://docs.qiime2.org/2017.7/data/tutorials/moving-pictures/table.qza
```

Comment:
These will need to become data.qiime2.org links or we will end up with a bootstrapping problem (the 2017.8 urls won't exist until the docs are built and the docs won't build until the 2017.8 urls exist)